load_template.php

```diff
@@ -51,22 +51,16 @@
 // Check that they have confirmed they wish to load the template.
 if ($confirm && confirm_sesskey()) {
     // First, remove all the existing elements and pages.
-    $sql = "SELECT e.*
-              FROM {customcert_elements} e
-        INNER JOIN {customcert_pages} p
-                ON e.pageid = p.id
+    $sql = "SELECT p.*
```

Comment:
This SQL seems to be just hanging around for no reason and is not used.

Reply:
Removed unused SQL.